docs/source/get_involved/code_of_conduct.rst

```diff
@@ -342,14 +397,161 @@ Guide <https://www.djangoproject.com/conduct/>`__ and we are also
+
+* any decisions that allocate resources primarily obtained due to the ``sktime`` project and the volunteer contributions of the community.
+  This includes research grants awarded directly for the benefit of the ``sktime`` project,
+  commercial opportunities arising from or directly directed to the ``sktime`` project,
+  and any activities that use the ``sktime`` brand as a primary branding.
+  This condition applies even if relevant decisions are made in personal roles, or roles other than ``sktime`` official roles.
+* any decisions by members of the ``sktime`` developer community, in their execution of an official ``sktime`` role.
+  This especially includes core developer, Community Council, and Code of Conduct Committee roles.
+
+Examples for in-scope:
+
+* an academic allocating grant monies from a grant, or a data science consultant providing consultancy services, with a primary ``sktime`` branding
+* hiring decisions for roles that are advertised with a primary ``sktime`` branding
+* spending decisions from ``sktime`` community administered accounts
+* decisions arising from academic collaboration or commercial requests directed to the ``sktime`` community,
+  e.g., via official email, or via digital ``sktime`` discussion tools (slack, GitHub discussion etc)
+
+"resources" in the above context are to be interpreted in the usual sense of the word,
+i.e., valuable possessions, opportunities, qualities, that can be accessed and allocated.
+
+Examples include but are not limited to:
+
+* budget, money, financial credit
+* job opportunities, business opportunities
+* decision making power, soft or hard influence
+* dedicated work time of subordinates or dependents
+* material possessions of value
+* information of value
+* access to any of the above
+
+Best practice
+^^^^^^^^^^^^^
+
+To ensure decision making in line with the code of conduct, decision making must be:
+
+* by the ``sktime`` community, through ``sktime`` community decision making mechanisms.
+  E.g., following decision making outlined in the ``sktime`` governance document.
+* in communication transparent to the ``sktime`` community, following communication requirements outlined in the ``sktime`` governance document.
+* pursuant of and in line with the charitable mission of ``sktime`` and NumFOCUS.
+
+Adherence to widely accepted guidelines of anti-corruption and anti-bribery practice
+(e.g., United Nations Anti-Corruption Guidelines) is strictly expected,
+especially for resource allocation decisions of major magnitude, e.g., in hiring processes,
+or decisions in the 5-digit dollar range and upwards.
+
+This strict requirement for best practice remains unchanged even if a local context may require less,
+e.g., university administrations,
+company policies, national anti-corruption and anti-bribery laws (or their absence), etc.
+
+For instance, it is in-principle possible to misappropriate ``sktime`` resource, while
```

Comment:
Just clarifying, under "resource" we mean our community funds, correct? If yes, fully agree

Reply:
yes, and other things, as clarified in the new appendix section "what is meant by resource" that @aiwalter has requested.

Reply:
Like said, I think that sktime has control only over the money that flows to sktime. Donations to OC, donations from industry, support from projects like GSoC and NumFOCUS and other forms of funds that come directly to the project.

Reply:
> I think that sktime has control only over the money that flows to sktime

Yes, I agree, in the sense that it also includes money that is *supposed to flow to sktime*, e.g., in the embezzlement scenario where someone redirects it maliciously.

Reply:
Yes agree. That includes the money that was supposed to go to the sktime community, like donations. A research grant is not an instance of such, for example, because a grant goes to the research, not the OSS.